flycheck.el

```diff
@@ -3187,6 +3197,13 @@ Return ERRORS, modified in-place."
           errors)
   errors)
 
+(defun flycheck-relevant-error-other-file-p (err)
+  "Determine whether ERR is a relevant error for another file."
+  (let ((file-name (flycheck-error-filename err)))
+    (and buffer-file-name file-name
+         (not (flycheck-same-files-p buffer-file-name file-name))
+         (eq 'error (flycheck-error-level err)))))
```

Comment:
Should we make the error level for other files customizable? It might be best to wait until the feature is requested, though.

Reply:
I agree we should wait for a request.